contracts/sumtree-orderbook/src/query.rs

```diff
@@ -214,3 +216,68 @@ pub(crate) fn denoms(deps: Deps) -> ContractResult<DenomsResponse> {
         base_denom: orderbook.base_denom,
     })
 }
+
+pub(crate) fn ticks_by_id(deps: Deps, tick_ids: Vec<i64>) -> ContractResult<TicksResponse> {
+    let ticks = tick_ids
+        .iter()
+        .map(|tick_id| {
+            let tick_state = TICK_STATE
+                .load(deps.storage, *tick_id)
+                .map_err(|_| ContractError::InvalidTickId { tick_id: *tick_id })
+                .unwrap();
```

Comment:
```suggestion
            let Some(tick_state) = TICK_STATE.may_load(deps.storage, *tick_id)? else {
                return ContractError::InvalidTickId { tick_id: *tick_id };
            };
```

to avoid unwrap and not gobbled up another querier errors